hack/configure-windows-ci.ps1

```diff
@@ -5,8 +5,9 @@ choco install --limitoutput --no-progress -y git golang
 
 #install containerd
 $version=$env:ctrdVersion
+echo "Installing containerd $version"
 curl.exe -L https://github.com/containerd/containerd/releases/download/v$version/containerd-$version-windows-amd64.tar.gz -o containerd-windows-amd64.tar.gz
-tar xvf containerd-windows-amd64.tar.gz
+tar.exe xvf containerd-windows-amd64.tar.gz
```

Comment:
?

Reply:
Not strictly necessary but makes it explicit we are using the tar.exe shipped with Windows vs a powershell alias.